### PR TITLE
virsh.py:set event cmd as variable in start_get_event func

### PR DESCRIPTION
**Pull request by nanli1**
def start_get_event needs to update event_cmd as param
Signed-off-by: nanli <nanli@redhat.com>

### CODE REVIEW

File: virttest/virsh.py [hunk 1]
```diff
@@ -642,15 +642,15 @@ def __str__(self):
...
         """
         Use a virsh session with subcommand 'event' to catch events
-
         :param vm_name: name of the vm to be catched
+        :param event_cmd: cmd to check event
         :return: the virsh session with 'event'
         """
         virsh_session = aexpect.ShellSession(VIRSH_EXEC)
-        event_cmd = 'event %s --all --loop' % vm_name
+        event_cmd = event_cmd % vm_name
```
Comment (Yingshun) [inline virttest/virsh.py:653]: I wonder if event_cmd need vm_name every time or not... Should it be event_cmd = event_cmd.format(vm_name)? What do you think?

Comment (nanli1) [reply virttest/virsh.py:653]: Hi yingshun ,Thanks for your patient. Could u please help to tell what's the difference `between "format" and "%s"` here. I find that if we use empty vm_name for "%s" type , we can also get ` event  --all --loop` @Yingshun

Comment (Yingshun) [reply virttest/virsh.py:653]: start_get_event() will be used by wait_event() and some virsh utilities will use this decorator, so I don't think it's reasonable to pass an empty vm_name here. The only difference between 'format' and '%' here is that there will no exception if no need to pass vm_name to 'event_cmd'.
For eg:
```
>>> event_cmd = "event --all --loop"
>>> my_vm = 'test1'
>>> event_cmd % my_vm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not all arguments converted during string formatting
>>> event_cmd.format(my_vm)
'event --all --loop'
>>> 
```

File: virttest/virsh.py [hunk 2]
```diff
@@ -642,15 +642,15 @@ def __str__(self):
 class EventTracker(object):
 
     @staticmethod
-    def start_get_event(vm_name):
+    def start_get_event(vm_name, event_cmd='event %s --all --loop'):
```
Comment (chunfuwen) [inline virttest/virsh.py:645]: event_cmd --> event_options, it looks like 'virsh event' is mandatory for any event action, and event_options e.g --all-loop is flexible, and will affect virsh event behaviour.

Comment (nanli1) [reply virttest/virsh.py]: Hi chunfu, Thanks  for your review. But we have met this event cmd ` "qemu-monitor-event vm_name --loop" `in some cases, so  'virsh event' may not be mandatory.

File: virttest/virsh.py (hunk 1 above)
Comment (nanli1) [reply virttest/virsh.py]: updated yingshun,Thanks for your review. Did I get your point?

**Comment by nanli1** [comment]
@chloerh hi haijiao,Could you please help re-review this pr ,This is related start_get_event and wait event :D

**Review by chloerh — APPROVED** [review]

**Review by Yingshun — APPROVED** [review]

**Review by chunfuwen — APPROVED** [review]
LGTM

**Comment by chunfuwen** [comment]
merged it since all reviewer approve this request

**Pull request merged**